tests/bsim/bluetooth/mesh/src/test_provision.c

```diff
@@ -1009,11 +1009,11 @@ static void test_provisioner_pb_remote_client_parallel(void)
 	/* scanning device with dev index 3 */
 	uuid[6] = '0' + 3;
 	uuid_to_provision_remote = uuid;
-	ASSERT_OK(bt_mesh_rpr_scan_start(&rpr_cli, &srv, uuid, 5, 1, &scan_status));
+	ASSERT_OK(bt_mesh_rpr_scan_start(&rpr_cli, &srv, uuid, 15, 1, &scan_status));
 	ASSERT_EQUAL(BT_MESH_RPR_SUCCESS, scan_status.status);
 	ASSERT_EQUAL(BT_MESH_RPR_SCAN_SINGLE, scan_status.scan);
 	ASSERT_EQUAL(1, scan_status.max_devs);
-	ASSERT_EQUAL(5, scan_status.timeout);
+	ASSERT_EQUAL(15, scan_status.timeout);
```

Comment:
Commit message says the scanner time is delayed, but there is scanning time in 3 times bigger than before.

Reply:
I extended it by 10 seconds as the unprovisioned beacon interval is 10 second so if it misses the first beacon, the second one it should receive.

Reply:
Should we use a factor of the MESH_INTERVAL define or a define derived from it? I.e instead of magic numbers?

Reply:
I meant the tests fall from time to time due to collisions when something new is added or changed. After this, some fixes with delays appear. Furthermore, the reason for them is forgotten after some time, and difficult to understand whether it is valid or not anymore.